test/parallel_api/ranges/std_ranges_memory.pass.cpp

```diff
@@ -0,0 +1,155 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/memory>
+#include <oneapi/dpl/ranges>
+
+#include "support/test_config.h"
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if _ENABLE_STD_RANGES_TESTING
+
+#include "std_ranges_memory_test.h"
```

Comment:
There is no much sense to to have a separate header file - it is small, and only on cpp file uses it.

Reply:
Separation between a head and a file mirrors the way `std_ranges_test.h` works with other tests, so let's keep it as is.